### DISCUSSION TITLE
Сделать управление ботом доступным только администраторам группы

### DISCUSSION THREAD

**Discussion by slavmetal** [m-messiah/xye-bot#37] [Ideas]
В группе, с добавлением бота, каждый начинает менять его режимы и кл-во пропускаемых сообщений, что неимоверно засоряет чат и, фактически, делает управление ботом невозможным.  Как вариант, можно разрешить боту реагировать только на администраторов группы.

**Comment by m-messiah** [comment]
Пока не придумал, как просто отличить админа от неадмина.
В приходящем апдейте от телеграмм такой инфы нет, значит нужно при получении команды спрашивать getChatMember и смотреть его статус. (+1 внешний запрос на команду)
Но если у группы нет администраторов, то для любого участника статус будет member, хотя на самом деле он и админ тоже. А значит нужен еще запрос на получение инфы о чате(+1): тип чата (пользователь, группа, супергруппа) и списка администраторов (+1) (с проверкой - там только создатель или есть кто-то еще). Итого получаем +3 запроса на слеш-команду. 

Будет решение проще - можно будет реализовать. В таком виде - как-то нерентабельно (хотя исходная проблема понятна и имеет место быть)
